src/patternfly/components/CodeEditor/code-editor.scss

```
@@ -71,9 +71,16 @@
 }
 
 .#{$code-editor} {
+  display: flex;
+  flex-direction: column;
+
   &.pf-m-read-only {
     --#{$code-editor}__main--BackgroundColor: var(--#{$code-editor}--m-read-only__main--BackgroundColor);
   }
+
+  &.pf-m-full-height {
+    height: 100%;
+  }
 }
 
```

Do we need styles for `#{$code-editor}__container`? IIRC from our convos, I'd expect it to be:

```suggestion
.#{$code-editor}__container {
  display: flex;
  flex-direction: column
  flex-grow: 1;
}
```

`display: flex; flex-direction: column;` - makes it a flex layout like the main code editor wrapper so that `__header` and `__main` are flex children, which makes this rule work https://github.com/patternfly/patternfly/blob/5006992758d980967cdcde91213692eda8cedeb9/src/patternfly/components/CodeEditor/code-editor.scss#L155

`flex-grow: 1;` - tells `__container` to fill the height of the code editor wrapper. Without this, the code editor won't fill the available space if you use `.pf-m-full-height` on the code editor wrapper.